FindChannelFunction/build.gradle

```diff
@@ -37,8 +41,10 @@ pmd {
 checkstyle {
     configFile = rootProject.resources.text.fromFile('config/checkstyle/checkstyle.xml').asFile()
     showViolations = true
+    toolVersion "8.30"
```

Comment:
Why do we set version?

Reply:
To make i compile... Checkstyle was not happy with default...